flare/ase/otf.py

```diff
@@ -186,8 +182,9 @@ def compute_properties(self):
             OTF structure object.
         """
 
-        # Reset FLARE calculator if necessary.
+        # Change to FLARE calculator if necessary.
         if not isinstance(self.atoms.calc, FLARE_Calculator):
+            self.flare_calc.results = {}
```

Comment:
A minor suggestion, maybe add a method in flare calculator.py called `reset()`, i.e.
```
def reset(self):
    self.results = {}
```

Just in case there could be some other parameters resetting needed in the future use

Reply:
Sure, I addressed this in 9cca1c2 and 6f6b0dc